Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -359,24 +350,6 @@ public final class DiscoveryPageViewModel: DiscoveryPageViewModelType, Discovery
         return sort == .magic && filterParams == DiscoveryViewModel.initialParams()
       }
 
-    let featureFlagValue = self.sortProperty.signal.skipNil()
```

Comment:
I'm not certain if the code under `// MARK: - Editorial Header` is safe to deprecate as well?

Reply:
Yeah I think that can go.